lsp/client-extension/node-env.nix

```diff
@@ -1,6 +1,6 @@
 # This file originates from node2nix
```

Comment:
Was this file automatically re-generated, or did you manual edit it and I need to review it too?

Reply:
This was all formatted by `nixpkgs-fmt`. But I don't think we should do this for generated nix files such as this. I'll add it to the ignore list and reverted the changes.